docs/05-technical-reference/00-architecture/ac-03-application-gateway.md

```diff
@@ -9,11 +9,11 @@ which are either unsecured or secured with various security mechanisms and prote
 
 ![Application Gateway Diagram](assets/ac-architecture-proxy-service.svg)
 
-<!-- TODO: mention that the path in Compass scenario contains API Bundle and API definition -->
+> **NOTE:** See how the [Gateway URL differs](../ac-01-application-gateway-details.md#application-gateway-url) for the [Standalone and Compass modes](../../01-overview/main-areas/application-connectivity/README.md).
 
 1. A Function calls Application Gateway. 
 2. Application Gateway extracts the Application name and the service name from the path. Using the extracted Application name, Application Gateway finds the respective Application custom resource and obtains the information about the registered external API, such as the API URL and security credentials. 
-3. Application Gateway gets a token from the OAuth server. The OAuth server can be additionally [secured with a client TLS certificate](../../03-tutorials/00-application-connectivity/ac-04-register-secured-api.md#register-an-oauth-20-mtls-secured-api) .
+3. Application Gateway gets a token from the OAuth server. The OAuth server can be additionally [secured with a client TLS certificate](../../03-tutorials/00-application-connectivity/ac-04-register-secured-api.md#register-an-oauth-20-mtls-secured-api).
```

Comment:
I am not sure it is proper to say that it is "additionally" secured with the cert. Those are basically two different flows - one uses user and secret, the other - user and certificate.
Since we focus on a single auth method here I would drop the whole second sentence.